### ISSUE TITLE
How to Add External Driver?

### ISSUE THREAD

**Issue by gsuresh2u** [FLYmaker/FLYF407ZG#34]
Hi, i am using 4 motors for Z axis (Nema23)
and i am using TMC5160 v1.2 for all 9 drivers and with 128 microstep
TMC5160 is capable of 4.5Amp, and My NEMA23 motors are 4A/phase
after starting 5 minutes of printing, my Z stepper drivers are heating so much, i am unable to touch them
here is my Stepper motor info
![Nema23-30 61Kg Motor Specs](https://user-images.githubusercontent.com/39871361/107465788-0136eb00-6b89-11eb-83e6-7b36454af20b.jpg)
and in Marlin advanced config, i modified as below for all Z motors
 ```
#if AXIS_IS_TMC(Z)
    #define Z_CURRENT       2545 //    4 Amp / 1.414 = 2.828 -> 2.828 * 0.9(90% Capacity) =2.545A = 2545mA
    #define Z_CURRENT_HOME  Z_CURRENT
    #define Z_MICROSTEPS     128
    #define Z_RSENSE         0.075   //0.11 (For TMC5160 its 0.075, i read it in some blog)
    #define Z_CHAIN_POS      -1
  #endif
```
Still having the same heat problem
Is there any way to add External Drivers like TB6600 ?

**Comment by jaysuk** [comment]
yes, you can run external drivers. just connect the step, enable and direction directly to the driver.
There are many instructions on the internet about how to do so
